### PR TITLE
AV-165201 Add Antrea support in IPv6

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

File: internal/lib/lib.go [hunk 1]
```diff
@@ -1913,3 +1913,41 @@ func IsChanClosed(ch <-chan struct{}) bool {
 	}
 	return false
 }
+
+func GetIPFromNode(node *v1.Node) (string, string) {
```
Comment (akshayhavile) [inline internal/lib/lib.go:1917]: who is calling this method?

Comment (akshayhavile) [reply internal/lib/lib.go:1917]: got it. we need to merge this after your earlier PR.

**Review by akshayhavile — APPROVED** [review]

**Review by swathinsankaran — DISMISSED** [review]

**Comment by swathinsankaran** [comment]
This PR should be merged after #1022.

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Review by swathinsankaran — APPROVED** [review]

File: internal/lib/lib.go [hunk 2]
```diff
@@ -1936,6 +1936,18 @@ func GetIPFromNode(node *v1.Node) (string, string) {
 				nodeV6 = strings.Split(nodeIP, "/")[0]
 			}
 		}
+		if GetCNIPlugin() == ANTREA_CNI {
+			if nodeIPstr, ok := node.Annotations["node.antrea.io/transport-addresses"]; ok {
```
Comment (akshayhavile) [inline internal/lib/lib.go:1940]: @aaha97 : Where can i get information about this annotation? will it be always used to indicate IPV4, V6  node ip?

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**